README.md

```diff
@@ -1,3 +1,9 @@
-# nodejs
+# examples/nodejs
+
+This repository contains an example node.js app configured for local development and deployment to Convox.
+
+The following files were added/modified against a brand new sample node.js app:
```

Comment:
Can probably drop the "brand new" here, the usage on rails was to imply it was freshly generated